charts/opentelemetry-demo/Chart.yaml

```diff
@@ -1,7 +1,7 @@
 apiVersion: v2
 type: application
 name: opentelemetry-demo
-version: 0.12.0
+version: 0.12.1
```

Comment:
Just merged in another PR, gotta bump this to `0.12.2` and then run `make generate-examples`.

Reply:
Ok , done, thank you . But I find this process a bit complex. Shouldn't we bump versions  on every release, and not on every pull  request?  Contributors  need to rebase constantly  just to fix version numbers.
Me I have contributed one character change in the documentation , but  the overhead of making it to master is quite high

Reply:
The process has become quite cumbersome as the charts have become more active. In the past, when there wasn't as much going on, it was a convenient way to ensure the latest charts were always released. 

Can you open an issue on this topic?